ext/drx/aarch64/scatter_gather.c

```diff
@@ -0,0 +1,53 @@
+/* **********************************************************
```

Comment:
Unless we expect additional files added to these subdirectories, maybe it would be simpler to use suffixes rather than subdirs, like we do for things like `core/unix/loader_{android,linux,macos}.c` and `ext/drsyms/drsyms_{dwarf,elf,macho}.c` and other places: so we'd have `ext/drx/scatter_gather_{aarch64,arm,riscv64,x86}.c`?